src/Controller/Component/SecurityComponent.php

```diff
@@ -510,12 +511,12 @@ protected function _debugPostTokenNotMatching(Controller $controller, $hashParts
             'Missing field \'%s\' in POST data'
         );
         $expectedUnlockedFields = Hash::get($expectedParts, 2);
-        $dataUnlockedFields = Hash::get($hashParts, 2) ?: [];
+        $dataUnlockedFields = Hash::get($hashParts, 2) ?: null;
```

Comment:
Why are you defaulting to `null` and then casting?

Reply:
Warning about string Operation on array in next line. Switching types on the same var here.

Reply:
Ok. I misread explode as implode derp.